### PR TITLE
drop-shadow

### PR DESCRIPTION
**Pull request by raji2004**

### CODE REVIEW

**Comment by raji2004** [comment]
fixed  the drop-shadow issue

**Comment by Balaguru4580** [comment]
Looks much nicer, and I appreciate the code formatting as well! Will merge now!

File: CSSfile.css [hunk 1]
```diff
@@ -71,53 +70,70 @@
...
 }
 
-.platform{
+.platform {
   width: 300px;
   height: 300px;
   z-index: -1;
-  scale:2;
+  scale: 2;
+  filter: drop-shadow(5px 5px 2px rgb(0, 0, 0));
```
Comment (Uzlopak) [inline CSSfile.css:97]: Crazy. I really hope you get the hacktoberfest T-Shirt. I mean this is ground breaking.

Comment (raji2004) [reply CSSfile.css:97]: thanks

**Pull request merged**